src/foundry/client/config.d.mts

```diff
@@ -1838,9 +1843,10 @@ declare global {
      * A mapping of status effect IDs which provide some additional mechanical integration.
      * @defaultValue `{ DEFEATED: "dead", INVISIBLE: "invisible", BLIND: "blind" }`
      */
-    specialStatusEffects: keyof CONFIG.SpecialStatusEffects extends never
-      ? CONFIG.DefaultSpecialStatusEffects
-      : CONFIG.SpecialStatusEffects;
+    specialStatusEffects: HandleEmptyObject<
+      InterfaceToObject<CONFIG.SpecialStatusEffects>,
+      CONFIG.DefaultSpecialStatusEffects
+    >;
```

Comment:
This technically covers a few more cases than it otherwise would and I intend to use this pattern everywhere it'll be relevant in CONFIG in #2593.